ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-master.yaml

```diff
@@ -0,0 +1,56 @@
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.20-openshift-4.14
+images:
+- dockerfile_path: Dockerfile
+  from: src
+  to: oadp-operator
+promotion:
+  name: oadp-operator
+  namespace: konveyor
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 1000m
+      memory: 512Mi
+tests:
+- as: unit-test
+  steps:
+    test:
+    - as: unit
+      commands: make test
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+- as: unit-test-post
+  postsubmit: true
+  steps:
+    test:
+    - as: unit
+      commands: make test submit-coverage
```

Comment:
how `submit-coverage` works?

Reply:
[Does this help?](https://github.com/openshift/oadp-operator/blob/78f6b7b3d26feeafd28b6dd0a164657435181168/Makefile#L274C15-L285)
```
submit-coverage:
	curl -Os https://uploader.codecov.io/latest/$(OS_String)/codecov
	chmod +x codecov
	./codecov > tmp.results 2> tmp.err
	cat tmp.results || echo "tmp.results not found"
	cat tmp.err || echo "tmp.err not found"
	@echo $$(cat tmp.results | grep 'resultURL' -c)
	@echo $$(cat tmp.err | grep 'please specify sha and slug manually' -c)
	if [ $$(cat tmp.err | grep 'please specify sha and slug manually' -c) == 1 ]; then \
		echo "specifying sha and slug manually" && ./codecov -C $(shell git rev-parse HEAD) -r openshift/oadp-operator; \
	fi
	rm -f codecov tmp.*
```

Reply:
https://docs.codecov.com/docs/codecov-uploader

Reply:
Would like to understand more about codecov. For example, here seems to be not very updated https://app.codecov.io/gh/openshift/oadp-operator (link from your README) and how to see results, when running locally. To see if you also add for pre-submit jobs